src/globus_sdk/login_flows/command_line_login_flow_manager.py

```diff
@@ -2,8 +2,10 @@
 
 import textwrap
 import typing as t
+from contextlib import contextmanager
```

Comment:
Ultra-minor style note, but I'll include it since I'm leaving comments already:

I prefer to import stdlib modules by name, not using `from ... import ...` syntax. This results in the fully qualified name being seen at the usage sites, which I think is beneficial, given how often we're reading code without the benefits of fancy IDE tools (e.g. if you're a crusty old man like me, and you don't use a fancy IDE).

So my style is generally to `import contextlib` and then use `@contextlib.contextmanager` where it's needed.

There are various counterarguments to this style, but I've personally never been convinced. I think `contextlib.contextmanager`, `textwrap.dedent`, `datetime.datetime`, and `socket.error`[^1] are less ambiguous than `contextmanager`, `dedent`, `datetime`, and `error`.

[^1]: `socket.error` is 100% me cheating to make a point. That's an alias for `OSError`. :grin: 

---

You'll see both styles used in the SDK, so you haven't violated a style rule. This is a _personal_ preference, and one that I haven't felt any need to force on my teammates or to battle-it-out until we reach consensus.

Reply:
I'd request we don't put this on someone external to the team.

As you mention it's not an agreed upon style guide within the SDK package itself, better to hold off on pushing our personal preferences until we can come to a consensus on them as maintainer preferences.

Reply:
Agreed; this wasn't even meant as a request for change. I wanted to share my perspective, note that there's no enforced style in the SDK, and leave it at that.